### DISCUSSION TITLE
#54 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0xPabloLI** [tog-ETH-er/togETHer#38] [General]
Agenda:

-Community updates
-Website update review
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：
https://meet.google.com/uin-uwpc-gta

**Comment by 0xPabloLI** [comment]
### PlanckerDAO

12.24 [读 Vitalik: What in the Ethereum application ecosystem excites me](https://plancker.notion.site/19-V-6-0a8282cc7e334c07bf91909153a64487)
12.18 [社交恢复 guardian 篇](https://plancker.notion.site/18-guardian-1fe9d7002261487d81fa5bf4d3006cc4)
12.15 [Crypto Tech Night跨社区分享：Account Abstraction and implementation: EIP4337](https://plancker.notion.site/Account-Abstraction-and-implementation-EIP4337-1950e6920397439fae3f83d41ca344e0)

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

9个PR #899 ～ #908

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《DAO openzeppelin governance搭建操作交流》 by 皓
  - https://www.youtube.com/watch?v=AZQeCFvgFCA

- 《FileCoin技术与生态介绍》by zhangxin

**Comment by Stamp9** [comment]
### 原语里弄

《RBF 原理及发展调研》 课题研究开始。

**Comment by xrdavies** [comment]
#### Rebase 社区

* 举办 2022 Rebase Hackathon
  - 140 人报名
  - 35 支队伍
  - 32 个完赛项目
  - 6 个大赛奖
  - 一个创意奖、一个最受欢迎奖
  - 3 个特别赞助奖
  - 7 场 Workshop
  - 48 小时分享，19 个分享主题

* 14 篇 Web3 极客日报

* 研报翻译一篇，（未完待续）
